packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -31,10 +31,10 @@ export const LAMBDA_TIMEOUT_MS = 9500
 export const TTL_DELTA = 60 * 60 * 24 * 365 * 2 // Keep records for 2 years
 export const logger = new Logger({serviceName: "updatePrescriptionStatus"})
 
-// AEA-4317 - Env vars for INT test prescriptions
+// AEA-4317 (AEA-4365) - Env vars for INT test prescriptions
 const INT_ENVIRONMENT = process.env.ENVIRONMENT === "int"
-export const TEST_PRESCRIPTION_1 = process.env.TEST_PRESCRIPTION_1 ?? ""
-export const TEST_PRESCRIPTION_2 = process.env.TEST_PRESCRIPTION_2 ?? ""
+export const TEST_PRESCRIPTIONS_1 = (process.env.TEST_PRESCRIPTIONS_1 ?? "").split(",") || []
+export const TEST_PRESCRIPTIONS_2 = (process.env.TEST_PRESCRIPTIONS_2 ?? "").split(",") || []
```

Comment:
Is it worth adding a .trim() to handle spaces between prescription id's?
Currently "abc, def" would be split to ["abc", " def"].
I can imagine that causing a bug later down the line

Reply:
Trim has been added.